tests/src/python/CMakeLists.txt

```diff
@@ -367,6 +363,10 @@ if (WITH_DESKTOP)
 endif()
 
 if (ENABLE_PGTEST)
+  ADD_PYTHON_TEST(PyQgsImportIntoPostGIS test_processing_importintopostgis.py)
+  ADD_PYTHON_TEST(PyQgsQueryResultModel test_qgsqueryresultmodel.py)
+  ADD_PYTHON_TEST(PyQgsVectorFileWriter test_qgsvectorfilewriter.py)
```

Comment:
These two shouldn't be dependent on the postgres tests being enabled. You'll need to split the tests themselves into two seperate files so all the non postgres parts of the files can still be run.

Reply:
Done for PyQgsVectorFileWriter. On the other hand, all tests in PyQgsQueryResultModel needs PostGres

Reply:
Ah sorry, I was referring to PyQgsVectorLayerUtils

Reply:
thanks, It's done too